Src/AutoFixture/Kernel/FieldSpecification.cs

```diff
@@ -11,6 +12,7 @@ public class FieldSpecification : IRequestSpecification
     {
         private readonly Type targetType;
         private readonly string targetName;
+        private readonly IEqualityComparer<string> nameComparison;
```

Comment:
Given that the containing class is a `FieldSpecification`, what is it going to compare?